### PR TITLE
refactor: list view topicwise list

### PR DESCRIPTION
**Pull request by ShivamSinghal1**

### CODE REVIEW

File: codedigger/lists/tests/test_views.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        self.assertEqual(res.status_code, 200)
+    
+    def test_check_levelwise_ladder_all_ladder_view(self):
+        test_url = reverse('levelwise-ladder')
+        res = self.client.get(test_url)
+        self.assertEqual(res.status_code, 200)
+   
+   #same as above but with an authenticated user 
+    def test_auth_check_topicwise_list_all_lists_view(self):
+        test_url = "http://127.0.0.1:8000/lists/topicwise/list/testinglist_topicwise"
```
Comment (ShivamSinghal1) [inline codedigger/lists/tests/test_views.py]: Use reverse functions.

File: codedigger/lists/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+from .test_setup import *
```
Comment (ShivamSinghal1) [inline codedigger/lists/tests/__init__.py]: Is this required?

File: codedigger/lists/tests/test_views.py (hunk 1 above)
Comment (aaradhyaberi07) [reply codedigger/lists/tests/test_views.py]: so reverse('topicwise-list') + list_name ?

File: codedigger/lists/tests/__init__.py (hunk 2 above)
Comment (aaradhyaberi07) [reply codedigger/lists/tests/__init__.py]: Nope, I'll remove it

File: codedigger/lists/tests/test_views.py (hunk 1 above)
Comment (ShivamSinghal1) [reply codedigger/lists/tests/test_views.py]: Don't know, Have to go through documentation.

**Pull request merged**